aptos/contracts/sources/i64.move

```diff
@@ -0,0 +1,99 @@
+module pyth::i64 {
+    use pyth::error;
+        
+    /// As Move does not support negative numbers natively, we use our own internal
+    /// representation.
+    struct I64 has copy, drop, store {
```

Comment:
this is clearly necessary since move doesn't have i64, but let's be careful with how we use this because the bool/unsigned representation has some weird properties (specifically two zeros). We should be clear in the docs how zero is represented, and then let's try not to implement arithmetic ops on this struct.

Reply:
Let's also explain in the comments on get_price etc. in the file below how we expect people to consume the I64's returned by the various functions. e.g., we expect them to call `get_magnitude_if_positive` for the price

Reply:
I've put in a check in the constructor to make it impossible to change `(0, true)` to `(0, false)`, so we will never have `(0, true)` values.